src/Arcus.Templates.Tests.Integration/Fixture/WebApiProjectOptions.cs

```diff
@@ -30,7 +29,17 @@ private WebApiProjectOptions(ProjectOptions options) : base(options)
         /// <summary>
         /// Creates a set of empty project options for the web API project; resulting in a default web API project when a project is created from these options.
         /// </summary>
-        public static readonly WebApiProjectOptions Empty = new WebApiProjectOptions();
+        public static WebApiProjectOptions Empty { get; } = new WebApiProjectOptions();
+
+        /// <summary>
+        /// Adds the project option to include an 'appsettings.json' file to the web API project.
+        /// </summary>
+        public WebApiProjectOptions WithIncludeAppSettings()
+        {
+            ProjectOptions optionsWithIncludeAppSettings = AddOption("--include-appsettings");
```

Comment:
General thought - Since we have long and short notations - Should we provide tests for both?

Reply:
Still nothing found on the issue. Seems like the short notations are just generated to unique combinations of the project option long name. So, with that in mind, it wouldn't make much sense to test something out of our hands.

Reply:
Ok